contracts/RNGBlockhash.sol

```diff
@@ -1,17 +1,12 @@
 // SPDX-License-Identifier: GPL-3.0
 
-pragma solidity ^0.6.6;
+pragma solidity 0.8.6;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/math/SafeMath.sol";
-import "@openzeppelin/contracts/utils/SafeCast.sol";
 
-import "./RNGInterface.sol";
+import "./interfaces/RNGInterface.sol";
```

Comment:
Keep in mind that this is a breaking change for any dependencies.  Keep the same, even if it makes you twitch!

Try to touch as little as possible!

Reply:
Done: https://github.com/pooltogether/pooltogether-rng-contracts/pull/22/commits/701badd75899625cd0a1f5bf3271b5c6aa5fa21e